docs/catalog.md

```diff
@@ -493,6 +493,9 @@ The following is an example of configuring Slack, creating a package binding, an
   $ wsk action invoke mySlack/post --blocking --result --param text 'Hello from OpenWhisk!'
   ```
 
+### Using the Slack token-based API
```

Comment:
@starpit  please add `token` to the list of parameters above

>The parameters are as follows:
url: The Slack webhook URL.
channel: The Slack channel to post the message to.
username: The name to post the message as.
text: A message to post.